tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTestBase.java

```diff
@@ -124,15 +112,15 @@ String checkStatusReadAndCloseResponse(Response.Status expectedStatus, Response
      * The started LRA will be named based on the class name and the running test name.
      */
     String lraClientId() {
-        return this.getClass().getSimpleName() + "#" + testName.getMethodName();
+        return lraTestService.lraClientId(this.getClass(), testName.getMethodName());
     }
 
     /**
      * Adjusting the default timeout by the specified timeout factor
      * which can be defined by user.
      */
     long lraTimeout() {
-        return Util.adjust(LraTckConfigBean.LRA_TIMEOUT_MILLIS, config.timeoutFactor());
+        return lraTestService.lraTimeout();
```

Comment:
Same comment for this method (ie it is only ever called once). Refactoring is normally to simplify/clean up code bit this change is increasing complexity.

Reply:
yeah, sorry for this. I guess I forgot about further refactoring later on.